introduction.md

```diff
@@ -0,0 +1 @@
+My name is Bob. I like pina coladas and walks in the rain.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.